### DISCUSSION TITLE
Using Molten-nvim - how?

### DISCUSSION THREAD

**Discussion by mdietrich16** [nix-community/nixvim#2138] [Q&A]
I am quite new to NixOS and installed molten-nvim in Nixvim, which works flawlessly. However, installing i.e. [jupyenv](https://github.com/tweag/jupyenv) for declarative Jupyter installation and invoke `:MoltenInit`, it does not find active kernels. I have found no way of installing jupyter that works with molten, but I am also very clueless about what would be needed or how it works.

**Comment by GaetanLepage** [comment]
This is probably because molten uses the python environment provided by nixvim which does not contain your installed `jupyenv`.
I know that people have successfully customized the python environment used by neovim.
You can use our `extraPython3Packages` option to add extra packages, but I'm not sure it will be enough in your case.

**Reply by taranarmo** [reply]
Is there a way to make molten plugin to use current shell environment? I use it with `notebook-navigator` and `iron` backend works okay but `molten` does not (e.g. can't find pandas set up via nix-shell).
